docs/source/AdministratorGuide/Systems/Transformation/index.rst

```diff
@@ -254,6 +257,15 @@ In order to use the ByJobType plugin, one has to:
           AutoAddedSites += LCG.RAL.uk
           AutoAddedSites += LCG.SARA.nl
           AutoAddedSites += LCG.RRCKI.ru
+          DataReconstruction
+          {
+            Exclude = ALL
+            Exclude += LCG.CERN.cern
```

Comment:
So, IIUC if you add ALSO the specific site, this will be excluded by the AutoAddedSites. May I say that this is a bit counter-intuitive? ("ALL" should include "LCG.CERN.cern" already)